include/fi_util.h

```diff
@@ -209,11 +209,20 @@ struct util_ep {
 	struct util_av		*av;
 	struct dlist_entry	av_entry;
 	struct util_eq		*eq;
+	/* CQ entries */
 	struct util_cq		*rx_cq;
 	uint64_t		rx_op_flags;
 	struct util_cq		*tx_cq;
 	uint64_t		tx_op_flags;
 
+	/* CNTR entries */
+	struct util_cntr	*tx_cntr; /* transmit/send */
+	struct util_cntr	*rx_cntr; /* receive       */
+	struct util_cntr	*rd_cntr; /* read          */
+	struct util_cntr	*wr_cntr; /* write         */
+	struct util_cntr	*rr_cntr; /* remote read   */
+	struct util_cntr	*rw_cntr; /* remote write  */
```

Comment:
nit: In a follow on patch, can you rename the last two to rem_rd_cntr, rem_wr_cntr?